spec/support/components/attachments/attachments_input.js

```diff
@@ -1,33 +1,55 @@
-var params = arguments;
+let params = arguments;
```

Comment:
Not sure why eslint isn't catching this file, but a lot of these `let`s can become `const` I think